test/TestProjects/MgmtMockAndSample/tests/Generated/Samples/Sample_DeletedManagedHsmCollection_RetrieveADeletedManagedHSM.cs

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Identity;
+using Azure.ResourceManager;
+using Azure.ResourceManager.Resources;
+
+namespace MgmtMockAndSample
+{
+    public partial class Sample_DeletedManagedHsmCollection_RetrieveADeletedManagedHSM
+    {
+        // Retrieve a deleted managed HSM
+        [NUnit.Framework.Test]
```

Comment:
Will those lines be removed when built into a real sample?

Reply:
Yes